packages/bbui/src/Form/DatePicker.svelte

```diff
@@ -14,11 +14,17 @@
   export let placeholder = null
   export let appendTo = undefined
   export let ignoreTimezones = false
-
+  export let range = false
   const dispatch = createEventDispatcher()
 
   const onChange = e => {
-    value = e.detail
+    if (range) {
+      // Flatpickr cant take two dates and work out what to display, needs to be provided a string.
+      // Like - "Date1 to Date2". Hence passing in that specifically from the array
+      value = e.detail[1]
```

Comment:
is there potential for this to be `undefined`?

Reply:
I don't believe so, when the range variable is passed to flatpickr it will always return an element in that position, but I will check it just to be sure, might be an edge case etc